apps/designer/package.json

```diff
@@ -22,6 +22,7 @@
     "@lexical/react": "^0.6.0",
     "@lexical/selection": "^0.6.0",
     "@lexical/utils": "^0.6.0",
+    "@prisma/client": "4.3.1",
```

Comment:
q: why is this needed in the `designer`? Shouldn't it reach to the `@prisma/client` only through `@webstudio-is/prisma-client`?

Reply:
For some other reasons it's likely that you have to use this in the `@webstudio-is/prisma-client`:
```prisma
generator client {
  output          = "../src/generated-client"
}
```

and perhaps that would fix your problems here. I have a draft for this change locally but didn't yet have time to finish it up.

Reply:
See, remix bundles all its packages because of esm. Though with pnpm all packages dependencies are no longer available in designer. So prisma/client was a check. So I decided to use hoisted until remix issues are solved.

Reply:
If we use custom output remix will try to bundle generated client